api/contextMenus/global_context_search/locales.js

```diff
@@ -1,19 +1,29 @@
-// Copyright 2017 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 // TLD: top level domain; the "com" in "google.com"
 export const tldLocales = {
-  'com.au': 'Australia',
-  'com.br': 'Brazil',
-  'ca': 'Canada',
-  'cn': 'China',
-  'fr': 'France',
-  'it': 'Italy',
-  'co.in': 'India',
-  'co.jp': 'Japan',
-  'com.ms': 'Mexico',
-  'ru': 'Russia',
-  'co.za': 'South Africa',
-  'co.uk': 'United Kingdom'
+  'com.au' : 'Australia',
+  'com.br' : 'Brazil',
+  'ca' : 'Canada',
+  'cn' : 'China',
+  'fr' : 'France',
+  'it' : 'Italy',
+  'co.in' : 'India',
+  'co.jp' : 'Japan',
+  'com.ms' : 'Mexico',
+  'ru' : 'Russia',
+  'co.za' : 'South Africa',
+  'co.uk' : 'United Kingdom'
```

Comment:
Throughout this PR I noticed an extra space added to the object literals. I've never seen javascript formatted this way 😕 . 

Should we re-evaluate this formatting rule?

Here's a few javascript syntax examples:
[Object initializer - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#syntax)
[Whitespace · Styleguide JavaScript](https://arcticicestudio.github.io/styleguide-javascript/rules/whitespace.html#inside-object-literal-properties)